src/helperFunctions/install.py

```diff
@@ -239,3 +208,42 @@ def load_main_config() -> configparser.ConfigParser:
         raise InstallationError('Could not load config at path {}'.format(config_path))
     config.read(str(config_path))
     return config
+
+
+def run_cmd_with_logging(cmd: str, raise_error=True, shell=False, **kwargs):
+    """
+    Runs `cmd` with subprocess.run, logs the command it executes and logs
+    stderr on non-zero returncode.
+    All keyword arguments are passed to subprocess.run.
+    """
+    logging.info(f"Running: {cmd}")
+    try:
+        # Split with shlex to have shell commands work as intendet
```

Comment:
Small typo (but I don't know if the comment is really needed -- I think it is pretty self-explanatory)
```suggestion
        # Split with shlex to have shell commands work as intended
```